test/runtests.jl

```diff
@@ -2,11 +2,10 @@ using UnitfulAngles
 using Unitful
 using Base.Test
 
-units = (u"turn", u"halfTurn", u"quadrant", u"sextant", u"octant", u"clockPosition", u"hourAngle", u"compassPoint", u"hexacontade", u"brad", u"°", u"grad", u"arcminute", u"arcsecond", u"rad", u"diameterPart")
-quantities = (1, 2, 4, 6, 8, 12, 24, 32, 60, 256, 360, 400, 21600, 1296000, 2π, 120π)
+units = (u"turnturn", u"turn", u"halfTurn", u"quadrant", u"sextant", u"octant", u"clockPosition", u"hourAngle", u"compassPoint", u"hexacontade", u"brad", u"°", u"grad", u"arcminute", u"arcsecond", u"rad", u"diameterPart")
+quantities = (1, 2, 4, 8, 12, 16, 24, 48, 64, 120, 512, 720, 800, 43200, 2592000, 4π, 240π)
```

Comment:
What is the purpose of the change in these numbers?

Reply:
I wanted to include turnturn in the units testing.  Doing so adds one more factor of 2 into the way you designed the test.

Reply:
I don't know what I mistyped -- the test worked for me ... so I copied improperly

Reply:
Sorry, but wouldn't it be better to just put a `0.5` first in `quantities`?